src/renderer/chrome-api.js

```diff
@@ -140,7 +141,7 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
   subscribeAndForwardEvents(chrome);
 
   if (!isPackaged) {
-    chrome = loggingProxy(chrome);
+    // chrome = loggingProxy(chrome);
```

Comment:
?

Reply:
even in dev env, we don't want logs by default, it's the entry point for log all API calls